tests/src/FunctionalJavascript/CiviCrmTestBase.php

```diff
@@ -28,14 +28,36 @@ protected function setUp() {
     $this->drupalPlaceBlock('local_actions_block');
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function cleanupEnvironment() {
+    parent::cleanupEnvironment();
+    $civicrm_test_conn = Database::getConnection('default', 'civicrm_test');
+    // Disable foreign key checks so that tables can be dropped.
+    $civicrm_test_conn->query('SET FOREIGN_KEY_CHECKS = 0;')->execute();
+    $civicrm_schema = $civicrm_test_conn->schema();
+    $tables = $civicrm_schema->findTables('%');
+    foreach ($tables as $table) {
+      if ($civicrm_schema->dropTable($table)) {
+        unset($tables[$table]);
+      }
+    }
+    $civicrm_test_conn->query('SET FOREIGN_KEY_CHECKS = 1;')->execute();
+  }
```

Comment:
I'm going to cherry-pick this into a new PR. Turns out we were not dropping any of the `civicrm_` tables between tests.